packages/backend/src/index.ts

```diff
@@ -1,6 +1,6 @@
-import { rootHttpRouterServiceFactory } from '@backstage/backend-app-api';
 import { statusCheckHandler } from '@backstage/backend-common';
```

Comment:
@davidfestal @gashcrumb can you please help me fix these deprecations?

Reply:
Yep, this line looks like the export has been moved and should now be:

```
import { rootHttpRouterServiceFactory } from '@backstage/backend-defaults/rootHttpRouter'
```

And probably we have to be careful what `@backstage/backend-defaults` version is pulled in from this point on.

Reply:
@gashcrumb what about `import { statusCheckHandler } from '@backstage/backend-common';`?

Reply:
Hmm, it looks like `statusCheckHandler` is going to be removed in favor of the ["Root Health Service"](https://backstage.io/docs/backend-system/core-services/root-health/) which I think we will need to this code to use instead.